docs/includes/pre-install-hardware-required.txt

```diff
@@ -1,12 +1,13 @@
 * 2 computers with memory and hard drives to use as the SecureDrop servers.
 * Mouse, keyboard, monitor (and necessary dongle or adapter) for
   installing the servers.
-* Dedicated physical computers for the Admin, Journalist, and Secure Viewing
-  Station that can boot to Tails. At *minimum* this is 2 computers.
+* Dedicated physical computers for the *Admin Workstation*, *Journalist
```

Comment:
Do you mean that the SVS should use a dedicated physical computer? It's unclear to me which computers would be used for which

Reply:
I don't think the old language was clear on this point either, but I've tried to clarify it in d7f913bb67c9eb4ee82d3f056b88f69542a345dc.

Reply:
Additionally, the admin workstation needs an ethernet port or a dongle that has one, and the admin-ws and SVS need at least 2 USB ports, probably 3 if you count for times you need to insert yubikeys/transfer devices/backup devices.  This probably means purchasing a usb hub (with an ethernet port), if not 2 hubs (one with ethernet and one without), in order to have a dedicated device for the SVS and a device you can use with the the admin-ws.   

This will become more and more relevant as "old" computers repurposed for workstations move farther into Dongleland and have fewer onboard ports.

Reply:
re emkll's point: what about

"At least 2 computers that can boot to Tails: a dedicated physical computer for the *Secure Viewing Station*, and one or more computers for the *Admin workstation*/*Journalist workstation(s).*"